core/sql/generator/GenPreCode.cpp

```diff
@@ -11343,19 +11343,6 @@ short HbaseAccess::extractHbaseFilterPreds(Generator * generator,
        preds.advance(vid))
     {
       ItemExpr * ie = vid.getItemExpr();
```

Comment:
There are probably hundreds of places in the code where we could add such an assert, I don't think this place needs it more than others. We had a discussion with Eric a while back and decided that we wouldn't need to check for ANDs here; this code is just a left-over from a previous design.